include/deal.II/lac/arpack_solver.h

```diff
@@ -238,6 +246,17 @@ class ArpackSolver : public Subscriptor
   bool initial_vector_provided;
   std::vector<double> resid;
 
+  /**
+   * Real part of the shift
+   */
+  double sigmar;
```

Comment:
i will keep it as-is for now, just to make sure i don't break anything. Internal parts can always be refactored later.